parcels/particle.py

```diff
@@ -125,6 +126,8 @@ def __init__(self, lon, lat, grid, dt=3600., time=0., cptr=None):
         type(self).lon.initial = lon
         type(self).lat.initial = lat
         type(self).time.initial = time
+        type(self).trajectory.initial = grid.lasttrajectory
```

Comment:
I'm not sure the `grid` should be counting the IDs. We might just want a global module level ID counter with a fixed initial value?